src/components/Layout/Header/NavBar/KeepKey/ChangePassphrase.tsx

```diff
@@ -0,0 +1,66 @@
+import { Flex, FormControl, FormLabel, Spinner, Switch } from '@chakra-ui/react'
+import { useTranslate } from 'react-polyglot'
+import { AwaitKeepKey } from 'components/Layout/Header/NavBar/KeepKey/AwaitKeepKey'
+import { LastDeviceInteractionStatus } from 'components/Layout/Header/NavBar/KeepKey/LastDeviceInteractionStatus'
+import { SubmenuHeader } from 'components/Layout/Header/NavBar/SubmenuHeader'
+import { useKeepKey } from 'context/WalletProvider/KeepKeyProvider'
+import { useWallet } from 'hooks/useWallet/useWallet'
+
+import { SubMenuBody } from '../SubMenuBody'
+import { SubMenuContainer } from '../SubMenuContainer'
+
+export const ChangePassphrase = () => {
+  const translate = useTranslate()
+  const {
+    keepKeyWallet,
+    setHasPassphrase,
+    state: { hasPassphrase }
+  } = useKeepKey()
+  const {
+    state: { awaitingDeviceInteraction }
+  } = useWallet()
+
+  const handleToggle = async () => {
+    const currentValue = !!hasPassphrase
+    setHasPassphrase(!hasPassphrase)
+    await keepKeyWallet?.applySettings({ usePassphrase: !currentValue })
```

Comment:
possible `UnhandledPromiseRejection` error

Reply:
Handled in https://github.com/shapeshift/web/pull/1341/commits/f9fe9b56dee0d189360272b2cc4ae6e9a2d20f34